### ISSUE TITLE
Add GSWP3 driver dataset to PEcAn inputs/drivers list

### ISSUE THREAD

**Issue by serbinsh** [PecanProject/pecan#1871]
It would be good to add the GSWP3 driver dataset: GSWP3 3-Hourly Atmospheric Forcing: Temperature, Pressure, Winds, Humidity, and Downward Longwave Radiation.  
http://hydro.iis.u-tokyo.ac.jp/GSWP3/intro.html

This driver set is used with CLM5 and could be a useful driver set for other models as well.
http://www.cesm.ucar.edu/models/cesm2.0/land/

## Description
Add this dataset to the list of available PEcAn drivers.  

However, its not clear how to get these drivers without pulling them off, say, the CESM svn repo as with a typical CLM run.  It could be that those with the data locally could use the drivers or those with access to the SVN.

## Context
These driver data would be used with CLM and other model runs in PEcAn

## Possible Implementation
May have to write met tool wrappers to work with the data when available locally or via the CESM svn (if the user has access).

**Comment by ankurdesai** [comment]
Agree, can you point me to where GSWP3 data lives? We tried to look for it during downscaling project but didn't get far with inquiries or searching.

**Comment by serbinsh** [comment]
@ankurdesai only way I was able to get the data was from the CCSM/CESM svn repository which is used to grab inputs for CLM models.  I could grab the complete dataset and share with you if thats helpful?  Not sure what the usage restrictions are however so maybe I need to look into that first?

**Comment by serbinsh** [comment]
@ashiklom @bpbond @mdietze @ankurdesai 

Yesterday at ORNL I talked with Forrest about wanting access/how to get access to the raw GSWP3 driver dataset so we can create model drivers in PEcAn.  He mentioned he would inquire about FTP access to the data (non CLM-afied version) this week.  I hope to bring good news.
